src/components/transactions/VotingInfo.vue

```diff
@@ -82,10 +85,48 @@ export default {
       this.setVotesFromFedMembers()
     },
   },
-  created() {
-    this.setVotesFromFedMembers()
+  async created() {
+    try {
+      await this.setVotesFromFedMembers()
+    } catch (error) {
+      console.error(error)
+    }
   },
   methods: {
+    getParamsByTokenType(decodedEvent, event) {
+      switch (this.globalState.currentTokenType) {
+        case TOKEN_TYPE_ERC_20:
+          return [
+            decodedEvent._tokenAddress,
+            decodedEvent._from,
+            decodedEvent._to,
+            decodedEvent._amount,
+            event.blockHash,
+            event.transactionHash,
+            event.logIndex,
+          ]
+        case TOKEN_TYPE_ERC_721:
+          return [
+            decodedEvent._originalTokenAddress,
+            decodedEvent._from,
+            decodedEvent._to,
+            decodedEvent._tokenId,
+            event.blockHash,
+            event.transactionHash,
+            event.logIndex,
+          ]
+        default:
```

Comment:
maybe throw an error in this case? since it would be an unsupported token type, we shouldn't know what to do